### PR TITLE
Fake timestamps in tests.

### PR DESCRIPTION
**Pull request by MicahZoltu**

### CODE REVIEW

File: test/index.ts [hunk 1]
```diff
@@ -178,6 +179,22 @@ it('UniswapOracle on-chain hashes', async () => {
...
+	const now = Math.round(new Date().getTime() / 1000)
+	const rpc = await createMemoryRpc(jsonRpcEndpoint, gasPrice)
+	const selfAddress = await rpc.addressProvider()
+
+	await fetch(`http://localhost:12340/${now}`)
+	await rpc.sendEth(selfAddress, 0n)
+	const firstBlock = await rpc.getBlockByNumber(false, 'latest')
+	expect(firstBlock!.timestamp).toEqual(new Date(now * 1000))
+
+	await fetch(`http://localhost:12340/${now + 1}`)
```
Comment (epheph) [inline test/index.ts]: Isn't this the likely behavior without faketime?

**Review by epheph — APPROVED** [review]
Left question about test, but looks good. Using 1 volume for all nodes is neat.

File: test/index.ts (hunk 1 above)
Comment (MicahZoltu) [reply test/index.ts]: Good point, I'll update it to match the mechanism we discussed, by taking the last block time and adding a bunch of seconds to it.

Comment (MicahZoltu) [reply test/index.ts]: Fixed.

**Pull request merged**